static/js/main.js

```diff
@@ -423,8 +423,25 @@ function fetchVersion() {
         });
 }
 
+// Initialize hotkey functionality.
+function initializeHotkeys() {
+    document.addEventListener('keydown', (event) => {
+        if (event.code === 'KeyB') {
+            toggleSelection();
+        } else if (event.code === 'KeyF') {
+            captureFrame();
+        } else if (event.code === 'KeyS') {
+            save();
+        } else {
+            pass
+        }
+        // TODO Add arrow keys for frame step in video.
```

Comment:
I would remove this comment since it is not as trivial as it makes it seem.

The comment just says to add hotkeys for that functionality, but we have no code already setup for that functionality.
So if there was a TODO, it should actually be at the top of the file and mention both the functionality and the hotkey.
But in this project, we just use issues for something of that magnitude.

As a side note, TODO comments should be formatted as: `TODO(developer): XXX.`.
E.g., `TODO(eriq): Do this stuff.`
Accepted initials are also fine instead of names/usernames.